OrangebeardLogger.cs

```diff
@@ -281,24 +312,21 @@ private string DescriptionForCurrentContainer()
 
         private void UpdateTestrunWithSystemInfo(string message)
         {
-            var launchInfo = _orangebeard.Launch.GetAsync(_launchReporter.Info.Uuid).GetAwaiter().GetResult();
-
-            var attrs = new List<ItemAttribute>();
             var launchAttrEntries = message.Split(new[] {"\r\n", "\r", "\n"}, StringSplitOptions.None);
 
-            foreach (var entry in launchAttrEntries)
-            {
-                var attr = entry.Split(new[] {": "}, StringSplitOptions.None);
-                if (attr.Length > 1 &&
-                    !attr[0].Contains("displays") &&
-                    !attr[0].Contains("CPUs") &&
-                    !attr[0].Contains("Ranorex version") &&
-                    !attr[0].Contains("Memory") &&
-                    !attr[0].Contains("Runtime version"))
-                    attrs.Add(new ItemAttribute {Key = attr[0], Value = attr[1]});
-            }
-
-            _orangebeard.Launch.UpdateAsync(launchInfo.Id, new UpdateOrangebeardLaunchRequest {Attributes = attrs});
+            var attrs = (
```

Comment:
Hier misschien nog een comment bij om uit te leggen waarom deze checks op displays etc gedaan wordt?

Reply:
dat is die gekke link schrijfwijze... feitelijk geen check, maar hiermee filter je (beetje a la streams) een lijst. Hij neemt alleen die entries mee die voldoen aan die criteria. Misschien is hier een contains alleen niet zo mooi (zal auto-generated zijn), mag best equals (/i) zijn

Reply:
ik laat deze voor nu even zo. Is volledig Rx specifiek.